package.json

```diff
@@ -51,7 +51,7 @@
     "whatwg-fetch": "2.0.3"
   },
   "scripts": {
-    "start": "node frontend/scripts/start.js",
+    "start": "PORT=3000 node frontend/scripts/start.js",
```

Comment:
It is but it's also create-react-apps default. Foreman will pick another one, I'm mainly specifying it here since if the port is taken webpack will just pick the next available which can be confusing for people, Foreman's port is less important.